src/contexts/HUDContextProvider.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {
+  ReactNode,
+  createContext,
+  useContext,
+  useState,
+  useEffect,
+} from 'react';
+import {
+  Settings,
+  LogOut,
+  ArrowUpCircle,
+  ArrowDownCircle,
+} from 'react-feather';
+import {
+  RightSideContainer,
+  OptionsContainer,
+  InventoryContainer,
+  SlotContainer,
+  InventorySlot,
+} from '../components/HUD';
+import { State } from '../gameLogic/state';
+
+/** Não existe nenhum valor sendo passado nesse contexto por enquanto,
+ *  isso quer dizer que isso poderia ser um componente e não um contexto.
+ *  Fiz como contexto prevendo que vamos precisar expor alguns estados daqui
```

Comment:
Para o MVP ou num futuro tipo multiplayer?

Reply:
Talvez até pro MVP mesmo, to pensando que o timer entra aqui, e talvez a gente coloque esse timer em algum outro lugar por exemplo.